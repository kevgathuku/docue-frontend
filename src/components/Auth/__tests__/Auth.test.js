import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Auth from '../Auth';

describe('Auth', function() {
  const props = { history: {} };

  it('renders correctly', function() {
    const wrapper = shallow(<Auth {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
