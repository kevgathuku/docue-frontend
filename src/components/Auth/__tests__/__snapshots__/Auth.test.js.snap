// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth renders correctly 1`] = `
<div
  className="container"
>
  <br />
  <div
    className="row"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="container"
        >
          <div
            className="row card-panel hoverable"
          >
            <div
              className="col s12"
            >
              <ul
                className="tabs"
              >
                <li
                  className="tab col s4"
                >
                  <a
                    className="active blue-text"
                    href="#login"
                  >
                    Login
                  </a>
                </li>
                <li
                  className="tab col s4"
                >
                  <a
                    className="blue-text"
                    href="#signup"
                  >
                    Signup
                  </a>
                </li>
              </ul>
            </div>
            <div
              className="col s12"
              id="login"
            >
              <Connect(Login)
                history={Object {}}
              />
            </div>
            <div
              className="col s12"
              id="signup"
            >
              <Connect(SignupForm)
                history={Object {}}
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
